pystac/html/Macros.jinja2

```diff
@@ -0,0 +1,91 @@
+{% macro square(background_color='#E1E1E1', border_color='#9D9D9D') -%}
+    <div style="
+        width: 24px;
+        height: 24px;
+        background-color: {{ background_color }};
+        border: 3px solid {{ border_color }};
+        border-radius: 5px;
+        position: absolute;">
+    </div>
+{%- endmacro %}
+
+
+{% macro extra_fields(parent) -%}
+    {% if parent.extra_fields %}
+        {% for key, value in parent.extra_fields.items() %}
+            <tr><td style="text-align: left;"><strong>{{ key }}:</strong> {{ value }} </td></tr>
+        {% endfor %}
+    {% endif %}
+{%- endmacro %}
+
+
+{% macro links(parent) -%}
+    {% if parent.links|length > 0 %}
+        <details>
+            <summary style="margin-bottom: 10px; margin-top: 10px;">
+                <h4 style="margin-bottom: 0px; display: inline;">Links</h4>
+            </summary>
+            {% for link in parent.links %}
+                {{ link._repr_html_() }}
+            {% endfor %}
+        </details>
+    {% endif %}
+{%- endmacro %}
+
+
+{% macro items(parent) -%}
+    {% if parent.get_items()|list|length > 0 %}
+        <details>
+            <summary style="margin-bottom: 20px;">
+                <h4 style="margin-bottom: 0px; display: inline;">Items</h4>
+            </summary>
+            {% for item in parent.get_items() %}
+                {{ item._repr_html_() }}
+            {% endfor %}
+        </details>
+    {% endif %}
+{%- endmacro %}
+
+
+{% macro children(parent) -%}
+    {% if parent.get_children()|list|length > 0 %}
```

Comment:
Similar to comment above, this is going to result in (possibly) multiple network requests to resolve each child as a `STACObject`. We may want to use `get_child_links` and summarize the link info instead.